cheatsheets/JSON_Web_Token_Cheat_Sheet_for_Java.md

```diff
@@ -93,26 +93,26 @@ JWTVerifier verifier = JWT.require(Algorithm.HMAC256(keyHMAC)).build();
 DecodedJWT decodedToken = verifier.verify(token);
 ```
 
-## Token sidejacking
+## Token Sidejacking
 
 ### Symptom
 
-This attack occur when a token has been intercepted/stolen by a attacker and this one use it to gain access to the system using targeted user identity.
+This attack occurs when a token has been intercepted/stolen by an attacker and they use it to gain access to the system using targeted user identity.
 
-### How to prevent
+### How to Prevent
 
-A way to protect is to add "user context" in the token. User context will be composed by the following information:
+A way to prevent it is to add an "user context" in the token. An user context will be composed of the following information:
```

Comment:
This should be `a "user context"` rather than `an "user context"`. Same for the next sentence, should be `a user context`